packages/ui/src/components/TreeView/TreeView.tsx

```diff
@@ -25,6 +25,8 @@ const TreeViewItem = forwardRef<
     xPadding: number
     /** name of entity */
     name: string
+    /** hide the title/name of entity */
+    hideName?: boolean
```

Comment:
meant to add a note, sorry 

This is cluttered and unnecessary in this case since the title is the name

![screenshot-2024-09-30-at-10 43 54](https://github.com/user-attachments/assets/a0a4344d-dc3e-45d1-bdaa-d7b4ed9e2b7c)